plugins/modules/gitlab_user.py

```diff
@@ -28,7 +28,7 @@
   - Stef Graces (@stgrace)
 requirements:
   - python >= 2.7
-  - python-gitlab python module
+  - python-gitlab python module < 4.0.0
```

Comment:
The constants have been defined in `gitlab.const` since forever, see 1.0.0 in 2017:
https://github.com/python-gitlab/python-gitlab/blob/1.0.0/gitlab/const.py

I think it'd be better to simply import them from there rather than place artificial constraints. WDYT?

Reply:
Great, thanks !

Would you mind also fixing `gitlab_protected_branch` module, which also uses constants directly from `gitlab` package ?

Reply:
Missed it, but fixed it :)